proseco/guide.py

```diff
@@ -134,6 +134,9 @@ def run_search_stages(self):
         """
         cand_guides = self.meta['cand_guides']
         self.log("Starting search stages")
+        if len(cand_guides) == 0:
+            self.log("There are no candidates to check in stages.  Exiting")
+            return cand_guides
```

Comment:
One edit.  I just should have a better comment here before return the empty cand_guides as the empty 'selected' set.  Could also bump this if statement back up one level and have a conditional on bothering to get here.